docker/ci/Dockerfile

```diff
@@ -20,6 +20,10 @@ RUN apk --no-cache add bash clang git g++ make nodejs openjdk8 python perl \
     && cd $(go env GOPATH)/src/github.com/GoASTScanner/gas \
     && git reset --hard d30c5cde3613e9ba0129febda849e4d4df1d57cd \
     && go install github.com/GoASTScanner/gas
+    && curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly
```

Comment:
Hmm, I don't know if wercker does anything special to run as a different user. I suspect it'll have the same home directory as the dockerfile commands. (But I don't know for sure.)

- - -

Should we specify a particular commit or release of Rust here instead of nightly? As it is, we're going to get an arbitrary nightly snapshot (from whenever we build the docker image) and be using that until we update it again. Since it'll be effectively fixed anyway, maybe we should put here explicitly what version it is?